woocommerce/woocommerce
# Update woocommerce admin scripts to align more with turborepo

### All Submissions:






### Changes proposed in this Pull Request:

Removed some the package build commands that were part of the main WooCommerce Admin build scripts.
I also removed some other scripts that I didn't think were necessary anymore.

Partly addresses #33498 

### How to test the changes in this Pull Request:

1. Pull this branch and run `git clean -xdf && pnpm install` (FYI the clean method removes anything not committed from your local folder), you can also just pull a clean version down in a different folder.
2. Run `pnpm -- turbo run build` and make sure it builds successfully and that WooCommerce still runs fine (make sure to test the home screen).
3. Make a change in one of the packages WooCommerce Admin depends on (like `js/components` or `js/data`) and run `pnpm -- turbo run build`, it should re-build the package and also WooCommerce Admin (with potentially other packages that are dependent on the changed package). The change should reflect in the UI (could add a `console.log` to make checking this easier).
4. Run `pnpm run start --filter=woocommerce/client/admin` and make sure it still builds fine and watches both changes in `plugins/woocommerce-admin` and any of the `js` packages.

### Other information:




### FOR PR REVIEWER ONLY:
